test/CollapsibleMixinSpec.js

```diff
@@ -67,13 +70,13 @@ describe('CollapsibleMixin', function () {
 
   describe('from collapsed to expanded', function(){
     beforeEach(function(){
-      instance = ReactTestUtils.renderIntoDocument(
+      instance = render(
         <Component>Panel content</Component>
       );
     });
 
     it('Should have collapsing class', function () {
-      instance.setProps({expanded:true});
+      instance.renderWithProps({expanded:true});
```

Comment:
If this is using my helper I believe it needs to be `instance = instance.renderWithProps(...)` otherwise its working on the old value, interesting that this doesn't fail tho....